README.md

```diff
@@ -26,8 +26,7 @@ Create the client:
 import Replicate from "replicate";
 
 const replicate = new Replicate({
-  // get your token from https://replicate.com/account
-  auth: process.env.REPLICATE_API_TOKEN,
+  auth: "my api token", // defaults to process.env.REPLICATE_API_TOKEN
```

Comment:
I see you dropped the comment about finding the token on the site. I'm guessing that was intentional. Do you think it's easy enough for most people to find their token on the site, and this comment is just getting in the way?

Reply:
Not at all intentional! Thanks for calling this out 😄

Reply:
```suggestion
  // get your token from https://replicate.com/account
  auth: "my api token", // defaults to process.env.REPLICATE_API_TOKEN
```